modules/ROOT/partials/fragment-installing-operator-using-cli.adoc

```diff
@@ -0,0 +1,60 @@
+[start={next-step}]
+. Specify the namespace in which you want to install the Operator. You can create a new namespace or switch to an existing one.
+
+.. Create a new namespace if necessary:
++
+[source,bash,options="nowrap",subs="+quotes"]
+----
+$ kubectl create namespace  __<namespace-name>__
+----
+
+.. Switch to the namespace in which you want to install the Operator:
++
+[source,bash,options="nowrap",subs="+quotes"]
+----
+$ kubectl config set-context --current --namespace=__<namespace-name>__
+----
+
+. Create a CatalogSource in the `openshift-marketplace` namespace:
++
+[source,bash,options="nowrap",subs=attributes+]
+----
+$ kubectl apply -f examples/{example-directory}/00-cs.yaml
+----
```

Comment:
@pwright This is just for the upstream right?

Reply:
yes, just upstream.